app/main.py

```
@@ -1,3 +1,85 @@
+
+    def __repr__(self):
+        return f"Distance(km={self.km})"
+
+    # Операторы сравнения
+    def __eq__(self, other):
+        if isinstance(other, Distance):
+            return self.km == other.km
+        elif isinstance(other, (int, float)):
+            return self.km == other
+        return NotImplemented
+
+    def __lt__(self, other):
+        if isinstance(other, Distance):
+            return self.km < other.km
+        elif isinstance(other, (int, float)):
+            return self.km < other
+        return NotImplemented
+
+    def __le__(self, other):
+        if isinstance(other, Distance):
+            return self.km <= other.km
+        elif isinstance(other, (int, float)):
+            return self.km <= other
+        return NotImplemented
+
+    def __gt__(self, other):
+        if isinstance(other, Distance):
+            return self.km > other.km
+        elif isinstance(other, (int, float)):
+            return self.km > other
+        return NotImplemented
+
+    def __ge__(self, other):
+        if isinstance(other, Distance):
+            return self.km >= other.km
+        elif isinstance(other, (int, float)):
+            return self.km >= other
+        return NotImplemented
+
+    # Арифметические операторы
+    def __add__(self, other):
+        if isinstance(other, Distance):
+            return Distance(self.km + other.km)
+        elif isinstance(other, (int, float)):
+            return Distance(self.km + other)
+        return NotImplemented
+
+    def __iadd__(self, other):
+        if isinstance(other, Distance):
+            self.km += other.km
+        elif isinstance(other, (int, float)):
+            self.km += other
+        else:
+            return NotImplemented
+        return self
+
+    def __sub__(self, other):
+        if isinstance(other, Distance):
+            return Distance(self.km - other.km)
+        elif isinstance(other, (int, float)):
+            return Distance(self.km - other)
```

In the `__sub__` method, consider handling cases where the result might be negative. Depending on the context, you might want to raise an exception or handle it differently.